### DISCUSSION TITLE
kpanda/user-guide/workloads/pod-config/lifecycle/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2979] [Announcements]
# kpanda/user-guide/workloads/pod-config/lifecycle/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/workloads/pod-config/lifecycle/

**Comment by LLLzxx** [comment]
生命周期配置 -- 停止前命令 -- 含义 “业务排水” 描述错误

**Reply by lemontrees123** [reply]
已更正
